<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OboeWindow</class>
 <widget class="QMainWindow" name="OboeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OboeWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="SubscriptionsPage">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QListView" name="subscriptionsList"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="SearchPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLineEdit" name="searchEntry"/>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="searchStart">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QListView" name="listView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="PlaylistsPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListView" name="playlistsList"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="SummaryPage">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Summary Page</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="NowPlayingPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListView" name="listView_2"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="navPane">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="navSubscriptions">
         <property name="text">
          <string>Subscriptions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="navSearch">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="navPlaylists">
         <property name="text">
          <string>Playlists</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="navNowPlaying">
         <property name="text">
          <string>Now Playing</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QFrame" name="nowPlayingPane">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="jumpBack">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="pausePlay">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="jumpForward">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="skipCurrent">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="goToSubscriptions">
   <property name="text">
    <string>Go To Subscriptions Page</string>
   </property>
   <property name="toolTip">
    <string>Go to the subscriptions information page</string>
   </property>
  </action>
  <action name="goToSearch">
   <property name="text">
    <string>Go To Search</string>
   </property>
   <property name="toolTip">
    <string>Go to the search page</string>
   </property>
  </action>
  <action name="goToPlaylists">
   <property name="text">
    <string>Go To Playlists</string>
   </property>
   <property name="toolTip">
    <string>Go to the playlists information page</string>
   </property>
  </action>
  <action name="goToNowPlaying">
   <property name="text">
    <string>Go To Now Playing</string>
   </property>
   <property name="toolTip">
    <string>Go to the now playing screen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
